vsts/pipelines/copySdksFromProdToStorageAccount.yml

```diff
@@ -0,0 +1,39 @@
+variables:
+  group: Oryx
+  destinationStorageAccount: https://oryxsdkssandbox.blob.core.windows.net
+
+jobs:
+    - job: CreateReleaseTag
+      pool:
+        name: AzurePipelines-EO
+        demands:
+          - ImageOverride -equals AzurePipelinesUbuntu20.04compliant
+      steps:
+      - checkout: self
+        clean: true
+
+      - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+        displayName: 'Component Detection - OSS Compliance'
+        inputs:
+          ignoreDirectories: '$(Build.SourcesDirectory)/tests'
+
+      - task: UseDotNet@2
+        displayName: 'Use .NET Core sdk 3.1.x'
+        inputs:
+          version: 3.1.x
+
+      - task: ShellScript@2
+        displayName: 'Copy SDKs from the prod storage account to a destination storage account'
+        env:
+          DEV_STORAGE_SAS_TOKEN: $(DEV-STORAGE-SAS-TOKEN)
+          SANDBOX_STORAGE_SAS_TOKEN: $(SANDBOX-STORAGE-SAS-TOKEN)
+          PROD_STORAGE_SAS_TOKEN: $(PROD-STORAGE-SAS-TOKEN)
+        inputs:
+          scriptPath: ./vsts/scripts/copySdksFromProdToStorageAccount.sh
+          args:  ${{ variables.destinationStorageAccount }}
+      
+      - task: ShellScript@2
+        displayName: 'Test Prod storage account'
```

Comment:
Yeah probably dont need the step... I just kept it around to make sure that everything looks ok after the script runs. But I think that we would have to add sandbox tests to oryx if we wanted this to do something meaningful for that account as well, since this is just running oryx tests